### PR TITLE
Add pti-gpu-unitrace

### PR DESCRIPTION
**Pull request by ZzEeKkAa**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
!There is no tarball release!
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by oleksandr-pavlyk** [comment]
I do not object to being a maintainer of this package. `unitrace` is a useful utility for triaging offload activity of SYCL applications.

**Comment by ZzEeKkAa** [comment]
@conda-forge/help-c-cpp, ready for review!

**Comment by ZzEeKkAa** [comment]
@xhochy sorry to ping you directly, but I've seen that you are reviewing c-cpp recipes. Does anything needed to add to this repo and what the following steps to get it merged?

File: recipes/pti-gpu-unitrace/meta.yaml
Comment (xhochy) [inline recipes/pti-gpu-unitrace/meta.yaml:3]: You need a release before this can be merged.

File: recipes/pti-gpu-unitrace/meta.yaml
Comment (xhochy) [inline recipes/pti-gpu-unitrace/meta.yaml:29]: `pin_compatible` is only meant to be used in `run`. Please keep them plain

File: recipes/pti-gpu-unitrace/meta.yaml
Comment (xhochy) [inline recipes/pti-gpu-unitrace/meta.yaml:33]: All of these probably should have a run_exports and thus should not be listed explicitly here.

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/pti-gpu-unitrace/meta.yaml
Comment (ZzEeKkAa) [reply recipes/pti-gpu-unitrace/meta.yaml:33]: Unfortunetely no.

Without those pins I have warnings during the build time:
```
Packaging pti-gpu-unitrace
Packaging pti-gpu-unitrace-0.0.1-h07fc403_0
number of files: 13
   INFO: sysroot: '/home/yevhenii/.miniforge3/conda-bld/pti-gpu-unitrace_1716226117221/_build_env/x86_64-conda-linux-gnu/sysroot/' files: '['/home/yevhenii/.miniforge3/conda-bld/pti-gpu-unitrace_1716226117221/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/zone1970.tab', '/home/yevhenii/.miniforge3/conda-bld/pti-gpu-unitrace_1716226117221/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/zone.tab', '/home/yevhenii/.miniforge3/conda-bld/pti-gpu-unitrace_1716226117221/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/tzdata.zi', '/home/yevhenii/.miniforge3/conda-bld/pti-gpu-unitrace_1716226117221/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/right/Zulu']'
   INFO (pti-gpu-unitrace,bin/unitrace): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge/noarch::sysroot_linux-64==2.17=h4a8ded7_14
WARNING (pti-gpu-unitrace,bin/unitrace): Needed DSO lib/libze_loader.so.1 found in ['conda-forge/linux-64::level-zero==1.17.0=h297d8ca_0']
WARNING (pti-gpu-unitrace,bin/unitrace): .. but ['conda-forge/linux-64::level-zero==1.17.0=h297d8ca_0'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
   INFO (pti-gpu-unitrace,bin/unitrace): Needed DSO lib/libstdc++.so.6 found in conda-forge/linux-64::libstdcxx-ng==13.2.0=hc0a3c3a_7
   INFO (pti-gpu-unitrace,bin/unitrace): Needed DSO lib/libgcc_s.so.1 found in conda-forge/linux-64::libgcc-ng==13.2.0=h77fa898_7
   INFO (pti-gpu-unitrace,bin/unitrace): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge/noarch::sysroot_linux-64==2.17=h4a8ded7_14
WARNING (pti-gpu-unitrace,bin/libunitrace_tool.so): $RPATH/libxptifw.so not found in packages, sysroot(s) nor the missing_dso_whitelist.
.. is this binary repackaging?
WARNING (pti-gpu-unitrace,bin/libunitrace_tool.so): $RPATH/libOpenCL.so.1 not found in packages, sysroot(s) nor the missing_dso_whitelist.
.. is this binary repackaging?
WARNING (pti-gpu-unitrace,bin/libunitrace_tool.so): Needed DSO lib/libze_loader.so.1 found in ['conda-forge/linux-64::level-zero==1.17.0=h297d8ca_0']
WARNING (pti-gpu-unitrace,bin/libunitrace_tool.so): .. but ['conda-forge/linux-64::level-zero==1.17.0=h297d8ca_0'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
   INFO (pti-gpu-unitrace,bin/libunitrace_tool.so): Needed DSO lib/libstdc++.so.6 found in conda-forge/linux-64::libstdcxx-ng==13.2.0=hc0a3c3a_7
   INFO (pti-gpu-unitrace,bin/libunitrace_tool.so): Needed DSO lib/libgcc_s.so.1 found in conda-forge/linux-64::libgcc-ng==13.2.0=h77fa898_7
WARNING (pti-gpu-unitrace,bin/libunitrace_tool.so): $RPATH/libintlc.so.5 not found in packages, sysroot(s) nor the missing_dso_whitelist.
.. is this binary repackaging?
   INFO (pti-gpu-unitrace,bin/libunitrace_tool.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge/noarch::sysroot_linux-64==2.17=h4a8ded7_14
   INFO (pti-gpu-unitrace,bin/libunitrace_tool.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/ld-linux-x86-64.so.2 found in CDT/compiler package conda-forge/noarch::sysroot_linux-64==2.17=h4a8ded7_14
Fixing permissions
Packaged license file/s.
INFO :: Time taken to mark (prefix)
        0 replacements in 0 files was 0.02 seconds
fatal: No names found, cannot describe anything.
WARNING: Importing conda-verify failed.  Please be sure to test your packages.  conda install conda-verify to make this message go away.
```

And they are not added to the released package:

```
Loading channels: done
pti-gpu-unitrace 0.0.1 h07fc403_0
---------------------------------
file name   : pti-gpu-unitrace-0.0.1-h07fc403_0.conda
name        : pti-gpu-unitrace
version     : 0.0.1
build       : h07fc403_0
build number: 0
size        : 554 KB
license     : MIT
subdir      : linux-64
url         : file:///home/yevhenii/.miniforge3/conda-bld/linux-64/pti-gpu-unitrace-0.0.1-h07fc403_0.conda
md5         : 5cf84c599f7682c8330470ce9be5c506
timestamp   : 2024-05-20 17:34:16 UTC
dependencies:
  - __glibc >=2.17,<3.0.a0
  - libgcc-ng >=13
  - libstdcxx-ng >=13
```

File: recipes/pti-gpu-unitrace/meta.yaml
Comment (ZzEeKkAa) [reply recipes/pti-gpu-unitrace/meta.yaml:3]: Talking to the maintainer to get a release tag

File: recipes/pti-gpu-unitrace/meta.yaml
Comment (xhochy) [reply recipes/pti-gpu-unitrace/meta.yaml:34]: Then please make a PR or file an issue with the feedstock that they are missing run_exports and reference it here.

Comment (ZzEeKkAa) [reply recipes/pti-gpu-unitrace/meta.yaml]: I've fixed `level-zero-feedstock` and opened an issues on `intel-compilers-repack-feedstock`.

File: recipes/pti-gpu-unitrace/meta.yaml
Comment (ZzEeKkAa) [reply recipes/pti-gpu-unitrace/meta.yaml]: Got the release!

**Comment by ZzEeKkAa** [comment]
@xhochy I've applied all discussions, but was unable to trigger linter after force push.

**Comment by ZzEeKkAa** [comment]
@xhochy ping

**Review by xhochy — APPROVED** [review]

**Comment by ZzEeKkAa** [comment]
Thank you @xhochy !

**Pull request merged**
